cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,21 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
-  it('should provide an ability to log in', () => {
-    
+describe ('Sign In page', () => {
+    beforeEach(() => {
+      cy.visit('https://react-redux.realworld.io/#/login')
+    });
+    it('should provide an ability to log in', () => {
+  
+      cy.get('[placeholder="Email"]')
+        .type('tarrik2@qa.team')
```

Comment:
According to the mentor's style recommendations, we  should write commands in one line and without blank lines between them